xla/python/weakref_lru_cache_test.py

```diff
@@ -76,6 +76,37 @@ def Body():
       cache(wrkey, GilReleasingCacheKey())
     t.join()
 
+  def testAnotherMultiThreaded(self):
+    num_workers = 5
+    barrier = threading.Barrier(num_workers)
+    cache = xla_client.weakref_lru_cache(lambda: None, lambda x, y: y, 2048)
+
+    class WRKey:
+      pass
+
+    def worker_add_to_cache():
+        barrier.wait()
+        wrkey = WRKey()
+        for i in range(10):
+            cache(wrkey, i)
+
+    def worker_clean_cache():
```

Comment:
Same here. Please use Camel case for function names.
```suggestion
    def WorkerCleanCache():
```